app/services/locations_to_file_service.rb

```diff
@@ -12,20 +12,27 @@ def initialize(base_path: Rails.root.join('config', 'locations'))
 
```

Comment:
- LocationsToFileService should be deleted in a different PR so that we keep the history of the json files generation.

- The delivery_locations.json was updated manually with specific ids that figgy is using.